firefoxbookmarks/src/extension.cpp

```diff
@@ -380,16 +383,19 @@ void FirefoxBookmarks::Extension::setProfile(const QString& profile) {
         return;
     }
 
+
     // Get the correct absolute profile path
     QString profilePath = ( profilesIni.contains("IsRelative") && profilesIni.value("IsRelative").toBool())
             ? QFileInfo(d->profilesIniPath).dir().absoluteFilePath(profilesIni.value("Path").toString())
             : profilesIni.value("Path").toString();
 
     // Build the database path
     QString dbPath = QString("%1/places.sqlite").arg(profilePath);
+    QString tempDbPath = QString("%1/places_albert.sqlite").arg(profilePath);
```

Comment:
I'm not sure that this is a good location to put the copy.
IMHO a cache location that is owned by albert may be more suitable.

Reply:
makes sense. I used `Core::Plugin::cacheLocation();` to get this cache directory.

In linux, it's saving in `$HOME/.cache/albert/org.albert.extension.firefoxbookmarks/places_albert.sqlite`